server/views/pages/contacts/manage/contactSearch.njk

```diff
@@ -137,7 +137,7 @@
                 %}
                 
                 {% set contactNameHtml %}
-                    <a href="/prisoner/{{ prisonerDetails.prisonerNumber }}/contacts/add/mode/EXISTING/{{ journey.id }}?contactId={{ result.id }}" class="govuk-link govuk-link--no-visited-state bapv-result-row" data-qa="add-contact-{{ result.id }}-link">
+                    <a href="/prisoner/{{ prisonerDetails.prisonerNumber }}/contacts/add/mode/EXISTING/confirmation/{{ journey.id }}?contactId={{ result.id }}" class="govuk-link govuk-link--no-visited-state bapv-result-row" data-qa="add-contact-{{ result.id }}-link">
```

Comment:
I suggest to hold this change until you properly integrate into the journey otherwise it'll break. I'm surprised this didn't break the cypress tests (possibly the journey.mode being set allows it to work?)

Reply:
Changes reverted